lambdas/README.md

```diff
@@ -1,10 +1,8 @@
 # National Document Repository: Lambdas
 
-This part of the repository contains all our Lambda functions, currently these include...
+[<< Back To README.md](../README.md)
 
-- create_document_reference_handler
-- search_patient_details_handler
-- document_reference_search_handler
+This readme will run you throug the requirements and instaling your virutal env required to test locally.
```

Comment:
*spelling
through
installing (although maybe this should be "running of" or "installation of"